Formula/swagger-codegen@2.rb

```diff
@@ -23,13 +23,11 @@ class SwaggerCodegenAT2 < Formula
   depends_on java: "1.8"
 
   def install
-    # Need to set JAVA_HOME manually since maven overrides 1.8 with 1.7+
-    cmd = Language::Java.java_home_cmd("1.8")
-    ENV["JAVA_HOME"] = Utils.safe_popen_read(cmd).chomp
+    ENV["JAVA_HOME"] = Language::Java.java_home("1.8")
```

Comment:
According to the website openjdk@11 could be supported. They're not very clear about it.